lib/manageiq/ui/classic/engine.rb

```diff
@@ -1,7 +1,16 @@
 require 'rails/all'
-require 'sass-rails'
-require 'coffee-rails'
-require 'patternfly-sass'
+
+if ENV["RAILS_ENV"] != "production" || defined?(Rake)
+  require 'sass-rails'
+  require 'coffee-rails'
+  require 'patternfly-sass'
+else
+  require 'bootstrap-sass/engine'
+  require 'font_awesome/sass/rails/engine'
```

Comment:
@himdel @skateman Moving these two lines here is probably the more controversial change because this was originally a dependency inside of `patternfly-sass`.  I decided to do this to make the change to probably the more used `patternfly-sass` gem less ugly, and put the burden of the optimization on our end.

Wasn't sure of a good was to do that generically in that gem, but if you guys can think of a better way, I am all ears.